var/spack/repos/builtin/packages/cub/package.py

```diff
@@ -34,6 +34,7 @@ class Cub(Package):
     url      = "https://github.com/NVlabs/cub/archive/1.6.4.zip"
 
     version('1.6.4', '924fc12c0efb17264c3ad2d611ed1c51')
+    version('1.4.1', '74a36eb84e5b5f0bf54aa3df39f660b2')
 
     def install(self, spec, prefix):
-        copy_tree('cub', prefix.include)
+        copy_tree('.', prefix)
```

Comment:
This commit intends to do two things:

1. Fix the include path error. `copy_tree('cub', prefix.include)` will install header file `cub/cub.h` to `include/cub.h`. However, cub header is use by `#include <cub/cub.h>` in programming. So the previous installation will lead not `cub/cub.h not found error`. 
2. Add some useful materials (doc, examples) to the installation.

Glad to know Spack adds `prefix.include` to `CPATH` and perhaps I've benefit from it. I'll follow this convenetion to install `cub/cub.h` into `$PREFIX/include/cub/cub.h`.